parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -285,7 +285,15 @@ if $REBOOTREQUIRED; then
 else
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         # logs_to_events should not be run on & commands
-        /usr/lib/apt/apt.systemd.daily &
+        {{- if EnableUnattendedUpgrade }}
+        systemctl unmask apt-daily.service apt-daily-upgrade.service
+        systemctl enable apt-daily.service apt-daily-upgrade.service
+        systemctl enable apt-daily.timer apt-daily-upgrade.timer
+        {{- end }}
+        # this is the DOWNLOAD service
+        # meaning we are wasting IO without even triggering an upgrade 
+        # -________________-
+        systemctl restart apt-daily.service
```

Comment:
id didn't understand why this is restart rather than start but I'm just a systemd ignoramus.

Reply:
pretty much the same, afaik there's no cases where start is actually preferable to restart